commit e09712661cbcc327e3a8eaef5dad47cbc46cce73
Author: Oscar Andreasson <oscar.andreasson@pelagicore.com>
Date:   Tue Jan 2 16:09:39 2018 +0100

    ivi-controller.c: screen_remove_layer not supported
    
    screen_remove_layer is added in weston 3.0 which is not yet supported
    by yocto.
    
    Signed-off-by: Oscar Andreasson <oscar.andreasson@pelagicore.com>

diff --git a/weston-ivi-shell/src/ivi-controller.c b/weston-ivi-shell/src/ivi-controller.c
index 6d95c88..584b238 100644
--- a/weston-ivi-shell/src/ivi-controller.c
+++ b/weston-ivi-shell/src/ivi-controller.c
@@ -1185,7 +1185,7 @@ controller_screen_remove_layer(struct wl_client *client,
         return;
     }
 
-    lyt->screen_remove_layer(iviscrn->output, layout_layer);
+//    lyt->screen_remove_layer(iviscrn->output, layout_layer);
 }
 
 static void

